micronaut-camunda-bpm-feature/transactional-test-data-jdbc/src/test/java/info/novatec/micronaut/camunda/bpm/feature/test/MnJpaProcessEngineConfigurationTest.kt

```diff
@@ -0,0 +1,88 @@
+package info.novatec.micronaut.camunda.bpm.feature.test
+
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest
+import org.camunda.bpm.engine.RepositoryService
+import org.camunda.bpm.engine.delegate.DelegateExecution
+import org.camunda.bpm.engine.delegate.JavaDelegate
+import org.camunda.bpm.model.bpmn.Bpmn
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Executes the inherited tests with a JPA configuration. [PersistBookDelegate] replaces [DoNothingDelegate] and persists entities.
+ *
+ * @author Lukasz Frankowski
+ * @author Tobias Schäfer
+ */
+@MicronautTest(transactional = false)
+internal class MnJpaProcessEngineConfigurationTest : MnProcessEngineConfigurationTest() {
+    @Inject
+    lateinit var repositoryService: RepositoryService
+
+    @Inject
+    lateinit var bookRepository: BookRepository
+
+    @BeforeEach
+    fun deployProcessModel() {
+        val xml = Bpmn.convertToString(Bpmn.createProcess("SaveBookProcess")
+                .executable()
+                .startEvent()
+                .serviceTask().camundaDelegateExpression("\${saveBookDelegate}")
+                .endEvent().done())
+        repositoryService.createDeployment()
+                .addString("savebook.bpmn", xml)
+                .deploy()
+    }
+
+    @Test
+    public override fun testCommit() {
```

Comment:
mhm, all methods are public by default in kotlin. or is this needed because of the `override`? :thinking: . If it's not needed, you can remove it on all tests.
Besides, I think it's best practice in Junit5 to do the naming like this : `fun \`what is the test about\`` ()

Reply:
* thanks - I removed "public" for all methods
* I'll rename the methods when the parent project has been switched to Kotlin. Currently, we're overriding the methods so we cannot name them differently.